docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,628 @@
+1. In Xcode, open this project file:
+
+   ```
+   Development/UID2SDKDevelopmentApp/UID2SDKDevelopmentApp.xcodeproj
+   ```
+1. Run the **UID2SDKDevelopmentApp** app scheme.
+
+(**GWH__SW I'm not sure if this whole section should be the procedure, rather than just steps 1 and 2?**)
+
+When you've started the application, ensure the **Client Side** checkbox is checked, then enter an email or phone number, and it will make the following UID2 SDK API call that sends a request to the UID2 service to generate an identity and resulting token for the email/phone input:
+
+#### For Android:
+`UID2Manager.getInstance().generateIdentity()`
+
+#### For iOS:
+`UID2Manager.shared.generateIdentity()`
+
+Once the API call successfully completes, the application will display the resulting identity and persist it inside the UID2Manager class.
+
+The identity contains the generated UID2 advertising token value which can be retrieved using
+
+#### For Android:
+`UID2Manager.getInstance().getAdvertisingToken()`
+
+#### For iOS:
+`UID2Manager.shared.getAdvertisingToken()`
+
+You need this value when you are making an ad request, which will be explained in the rest of this guide.
+
+By default, the development application will use a default set of subscription ID and public key stored in:
+
+#### For Android:
+`com.uid2.dev.ui.MainScreenViewModel.Companion`
+
+#### For iOS:
+`RootViewModel`
+
+And connect to the UID2 integration environment as specified on:
+
+#### For Android:
+`com.uid2.UID2Manager.Companion#init`
+
+#### For iOS:
+`see UID2SDKDevelopmentApp/UID2SDKDevelopmentApp/Info.plist`
+
+You can change the credentials to ones assigned to you and connect to the UID2 Production environment if necessary. Read the Optional: Reduce Latency by Setting the API Base URL for the Production Environment section below.
+
+## Complete UID2 Account Setup
+
+Complete the UID2 account setup by following the steps described in the Account Setup page. As part of the account setup process, you'll need to provide a list of all application names (Android Application ID, iOS App Bundle ID and App Store ID) for the mobile applications that you'll be integrating with the UID2 Mobile SDKs.
+
+When account setup is complete, you'll receive a [subscription ID and public key](../getting-started/gs-credentials.md#subscription-id-and-public-key). These values are unique to you, and you'll be required to use them to configure the UID2 mobile SDK.
+
+## Add the UID2 Mobile SDK to Your Mobile Application
+
+to add the mobile SDK to your app, follow the applicable documentation:
+
+- [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md)
+- [UID2 SDK for iOS Reference Guide](../sdks/uid2-sdk-ref-ios.md)
+
+(**GWH__SW not sure what it means "Follow the instructions on a draft version of a new UID2 SDK for Android reference guide" please clarify.**)
```

Comment:
i meant the iOS/Android reference guides i have updated in this [PR](https://github.com/IABTechLab/uid2docs/pull/547)

Reply:
Hi @sunnywu I've updated but still not totally sure... seems to me that for instructions to add the mobile SDK to their app maybe they should go to the SDK doc, so I put that. The integration guides as you know are not broken out by Android/iOS plus this is an integration guide. Anyway I did what I thought was best and it will go to you for review.